backend-tests/tests/conftest.py

```diff
@@ -23,14 +23,17 @@
 from requests.packages import urllib3
 from testutils.common import wait_until_healthy
 from testutils.infra.container_manager.kubernetes_manager import isK8S
+from testutils.api.client import wait_for_port
 
 
 urllib3.disable_warnings()
 
 wait_until_healthy("backend-tests")
 
 
-forward_port = 8080
+def get_free_tcp_port() -> int:
```

Comment:
Shouldn't this be located somewhere in testutils (for example, testutils/util/network.py) so we don't replicate code?